samza-core/src/main/scala/org/apache/samza/metrics/ContainerProcessManagerMetrics.scala

```diff
@@ -52,6 +52,7 @@ class ContainerProcessManagerMetrics(
     val mFailedContainers = newGauge("failed-containers", () => state.failedContainers.get())
     val mReleasedContainers = newGauge("released-containers", () => state.releasedContainers.get())
     val mContainers = newGauge("container-count", () => state.containerCount)
+    val mInvalidNotifications = newGauge("invalid-notifications", () => state.invalidNotifications.get())
```

Comment:
From what I can tell, these notifications are not "invalid". They are "duplicate", "redundant" or "ignored", but not "invalid" 

I'd suggest a rename, but it's your call.